com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Added a new API to bake HDRP probes from C# (case 1276360)
+- Added support for raytracing for AxF material
+- Added area light moment shadows for AxF material
```

Comment:
it is not moment shadow but just exponential. Let's just say add area shadow support